### DISCUSSION TITLE
In which version will the performance of the harvester be improved?

### DISCUSSION THREAD

**Discussion by NTOHPN** [Chia-Network/chia-blockchain#11436] [Support]
The current harvester can not use too many Russian hard drives, the performance is very poor.

**Comment by loppefaaret** [comment]
Could you perhaps help me better understand exactly what is going on with the harvester on your system ? 
what version are you using ?
what OS ?
what kind of system setup ?
hardware ?

**Reply by NTOHPN** [reply]
There are 165 hard disks on the computer. After the harvester is started, the output computing power is very abnormal. There are many timeouts and invalid ones.
At the same time, I set the harvester to scan only 50 of them on this computer, and it will be normal.
The current problem is that the computer has too many disks, and the performance of the harvester cannot keep up.

**Comment by NTOHPN** [comment]
windows server 2022，chia1.3.4，cpu2650*2，64GBddr3

**Reply by loppefaaret** [reply]
how is the system behaving if you access more than 50 drives without doing it through Chia ?
Have you tried running without the GUI ?

**Reply by NTOHPN** [reply]
我目前使用hpool矿池提供的工具，用它访问我电脑上的165块硬盘，没有问题，平均读取延迟0.1秒左右。

**Reply by NTOHPN** [reply]
> 如果您不通过 Chia 访问超过 50 个驱动器，系统会如何运行？ 你试过在没有 GUI 的情况下运行吗？

使用GUI和单独使用收割机均会出现收割机不能正常运行的错误。
